web/packages/teleterm/src/mainProcess/rootClusterProxyHostAllowList.ts

```diff
@@ -77,22 +77,23 @@ export function manageRootClusterProxyHostAllowList({
 
     allowList.clear();
     for (const rootCluster of rootClusters) {
-      if (!rootCluster.proxyHost) {
-        continue;
+      if (rootCluster.proxyHost) {
+        let browserProxyHost: string;
+        try {
+          browserProxyHost = proxyHostToBrowserProxyHost(rootCluster.proxyHost);
+          allowList.add(browserProxyHost);
+        } catch (error) {
+          logger.error(
+            'Ran into an error when converting proxy host to browser proxy host',
+            error
+          );
+        }
       }
 
-      let browserProxyHost: string;
-      try {
-        browserProxyHost = proxyHostToBrowserProxyHost(rootCluster.proxyHost);
-      } catch (error) {
-        logger.error(
-          'Ran into an error when converting proxy host to browser proxy host',
-          error
-        );
-        continue;
+      // Allow the SSO hostname for SSO login/mfa redirects.
+      if (rootCluster.ssoHost) {
+        allowList.add(rootCluster.ssoHost);
```

Comment:
One thing I forgot about is that we should also call `proxyHostToBrowserProxyHost` on `rootCluster.ssoHost` as we do for proxy host above. This is in case `ssoHost` includes a default port of `443` (the details are explained in the comment for that function).

Of course it's far less likely that `ssoHost` is going to include it compared to `proxyHost` where Teleport adds `:443` all the time. But I suppose it's still worth including.